spec/support/backend.rb

```diff
@@ -0,0 +1,78 @@
+module BackendHelper
```

Comment:
mostly just copied out of promiscuous so we aren't coupled to promiscuous spec helpers